Runtime/Core/ApiGatewayManagement/IAmazonGameLiftWrapper.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace AmazonGameLiftPlugin.Core.ApiGatewayManagement
 {
-    public interface IAmazonGameLiftClientWrapper
+    public interface IAmazonGameLiftWrapper
```

Comment:
Sorry, I probably don't have context here. What's the reason this is being renamed? Is this because this includes methods that are used by the Editor as well as the Server?

I'm wondering if there's anyway to indicate which of these are used by what using comments, if the above is the reasoning

Reply:
comment added